modules/virt-about-vtpm-devices.adoc

```diff
@@ -10,9 +10,30 @@ A virtual Trusted Platform Module (vTPM) device functions like a
 physical Trusted Platform Module (TPM) hardware chip.
 You can use a vTPM device with any operating system, but Windows 11 requires
 the presence of a TPM chip to install or boot. A vTPM device allows VMs created
-from a Windows 11 image to function without a physical TPM chip.
+from a Windows 11 image to function without a physical TPM chip. A vTPM device also protects virtual machines by storing secrets without physical hardware.
```

Comment:
I don't think we should say this. I am not sure what it means, or if it is accurate. When I see secrets I immediately think kubernetes secrets which is not the case.

Reply:
Thank you, you are right that this is misleading and I wonder if we are entering the territory of trying to explain what a physical TPM device is. If this information is required, I think it would make more sense earlier in the paragraph right after the first sentence and we should be more specific about what certificates and private keys it is used for. @alromeros What do you think?

Reply:
> I think it would make more sense earlier in the paragraph right after the first sentence and we should be more specific about what certificates and private keys it is used for

I agree. If we want to include this sentence it makes more sense to have it earlier in the paragraph. I don't consider it a must though.

> When I see secrets I immediately think kubernetes secrets which is not the case.

This was my first thought too. In case we want to keep this sentence I'd try to say something more generic, such as "sensitive data" instead of secrets.

Reply:
Thank you. I removed it as it does not feel like we are doing justice to explaining what TPM devices are and I don't think it is the purpose of this section.